### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by Brave111** [sg-nm/Evolutionary-Autoencoders#1]
i want to know if change another dataset ,how do it ?can you briefly introduce it?

**Comment by sg-nm** [comment]
For both the inpainting and denoising tasks, you can use another dataset by changing the script "cnn_train.py" and "exp_main.py".
You need to specify the location of the dataset in the init function of cnn_train.py.
For example:

```
if dataset_name == 'XXX':
    data_transform = transforms.Compose([transforms.ToTensor()])
    test_data_transform = transforms.Compose([transforms.ToTensor()])
    dataset = dset.ImageFolder(root='location of dataset XXX/train', transform=data_transform)
    test_dataset = dset.ImageFolder(root='location of dataset XXX/val', transform=test_data_transform)
    self.dataloader = torch.utils.data.DataLoader(dataset, batch_size=self.batchsize, shuffle=True, 
       num_workers=int(num_work), drop_last=True)
    self.test_dataloader = torch.utils.data.DataLoader(test_dataset, batch_size=1, shuffle=False, 
       num_workers=int(num_work))
```

Then you need to specify the name "XXX" in exp_main.py:
`eval_f = CNNEvaluation(gpu_num=args.gpu_num, epoch_num=20, dataset='XXX', verbose=True, imgSize=imgSize, batchsize=16, mask=args.mask)`
